### PR TITLE
Inteli stage genesis file

### PR DESCRIPTION
**Pull request by mattdean-digicatapult**
Accompanies https://github.com/digicatapult/vitalam-flux-infra/pull/6

### CODE REVIEW

**Review by dblane-digicatapult — APPROVED** [review]

**Review by n3op2 — APPROVED** [review]

File: node/src/command.rs [hunk 1]
```diff
@@ -49,7 +49,8 @@ impl SubstrateCli for Cli {
     fn load_spec(&self, id: &str) -> Result<Box<dyn sc_service::ChainSpec>, String> {
         Ok(match id {
             "dev" => Box::new(chain_spec::development_config()?),
-            "" | "vitalam" => Box::new(chain_spec::vitalam_config()?),
+            "" | "inteli-stage" => Box::new(chain_spec::inteli_stage_config()?),
+            "vitalam" => Box::new(chain_spec::vitalam_config()?),
```
Comment (danielcooperxyz) [inline node/src/command.rs:53]: so our network runs all the chsins at once? or specify a single one?

Comment (mattdean-digicatapult) [reply node/src/command.rs:53]: specify one, that's in a `match` statement

**Review by danielcooperxyz — COMMENTED** [review]
why do we store genesis in the node repo and not the infra repo? is it compiled as part of the node binary?

**Comment by mattdean-digicatapult** [comment]
> why do we store genesis in the node repo and not the infra repo? is it compiled as part of the node binary?

yep, it get's compiled in. That way we can just refer to the chain by name

**Pull request merged**